app/components/post_header/index.js

```diff
@@ -19,8 +22,11 @@ function makeMapStateToProps() {
         const post = getPost(state, ownProps.postId);
         const commentedOnUser = getUser(state, ownProps.commentedOnUserId);
         const user = getUser(state, post.user_id) || {};
+        const currentUser = getCurrentUser(state);
         const teammateNameDisplay = getTeammateNameDisplaySetting(state);
         const militaryTime = getBool(state, Preferences.CATEGORY_DISPLAY_SETTINGS, 'use_military_time');
+        const enableTimezone = isTimezoneEnabled(state);
+        const userTimezone = enableTimezone && getUserCurrentTimezone(currentUser.timezone);
```

Comment:
What type does getUserCurrentTimezone return? If it's not a boolean, this should be wrapped with `Boolean` or else it'll cause prop type warnings

Reply:
`getUserCurrentTimezone` returns the user's currentTimezone.  the selector could return `user.automaticTimezone` or `user.manualTimezone`

Reply:
Making `userTimzone` a string

Reply:
If `enableTimezone` is false, then this will return a boolean. Can you change this so that its type is consistent?